### PR TITLE
Let notification_center send notifications for all highlights (including #channels)

### PR DESCRIPTION
**Pull request by Craige**
See title. `notification_center.py` doesn't currently send notifications for highlights in channels.

### CODE REVIEW

File: notification_center.py [hunk 1]
```diff
@@ -30,7 +30,7 @@
 	if not weechat.config_is_set_plugin(key):
 		weechat.config_set_plugin(key, val)
 
-weechat.hook_print('', 'irc_privmsg', '', 1, 'notify', '')
+weechat.hook_print('', ', '', 1, 'notify', '')
```
Comment (sindresorhus) [inline notification_center.py]: You have a typo in there. Make sure you test this change in Weechat, as I no longer use Weechat, so I can't test.

Comment (Craige) [reply notification_center.py]: Will fix. I have this running in wee-chat now, so I'm not sure how I missed that when I pushed. It's definitely correct locally now.

**Comment by sindresorhus** [comment]
Ping @Craige

**Comment by megalithic** [comment]
@sindresorhus / @Craige - can this be merged now?

**Comment by Craige** [comment]
@sindresorhus, @megalithic - I forgot about this. Think I still have this all locally and can fix it up.

**Comment by megalithic** [comment]
@Craige - thank you sir! that'd be fantastic.

**Comment by 7sDream** [comment]
stilllllllllllllllllll not push?

**Comment by 7sDream** [comment]
oh, I misunderstand this pr, your script works great, thank you!

**Comment by sindresorhus** [comment]
Bump

**Comment by sindresorhus** [comment]
Since this is stale, I'm moving it to an issue: https://github.com/sindresorhus/weechat-notification-center/issues/24

**Pull request closed without merging**